src/FE/src/app/pages/profiles-page/profiles-page.component.ts

```diff
@@ -203,14 +204,26 @@ export class ProfilesPageComponent implements AfterViewInit, OnInit {
   }
 
   async onDelete() {
-    const result = await this.profileService.deleteProfile(this.selectedRow?.profileId!)
-    if (result) {
-      this.datasource.data = this.datasource.data.filter((profile: Profile) => profile.profileId !== this.selectedRow?.profileId);
-      this.datasource._updateChangeSubscription();
-      this.snackbarService.openSnackBar(this.translateService.instant('SUCCESS_PROFILE_DELETED'), true);
-    } else {
-      this.snackbarService.openSnackBar(this.translateService.instant('ERROR_PROFILE_DELETED'), false);
-    }
+    const dialogRef = this.dialog.open(ConfirmDialogComponent, {
+      data: {
+        title: this.translateService.instant('CONFIRM_DELETE_PROFILE_TITLE'),
+        message: this.translateService.instant('CONFIRM_DELETE_MESSAGE') + this.selectedRow?.name + '?'
+      },
+      maxWidth: '15vw',
+      minWidth: '15vw',
+    });
+    dialogRef.afterClosed().subscribe(async (confirmed: boolean) => {
+      if (confirmed) {
+        const result = await this.profileService.deleteProfile(this.selectedRow?.profileId!)
+        if (result) {
+          this.datasource.data = this.datasource.data.filter((profile: Profile) => profile.profileId !== this.selectedRow?.profileId);
```

Comment:
kunne denne deles i to linjer? ved godt den var lang før, men nu hvor vi er forbi her